src/AlertBox/AlertBox.tsx

```diff
@@ -62,9 +62,9 @@ const TYPES = {
 };
 
 AlertBox.propTypes = {
-  children: PropTypes.node.isRequired,
-  className: PropTypes.string,
-  title: PropTypes.string.isRequired,
-  type: PropTypes.oneOf(_.values(TYPES)),
-  isClosable: PropTypes.bool,
+  children:  React.PropTypes.node.isRequired,
```

Comment:
if we're making these changes anyway, we might as well migrate our PropType references to `import * as PropTypes from "prop-types";`